### PR TITLE
Activate api-based Toolshed interface in Admin panel

### PR DESCRIPTION
**Pull request by guerler**
This PR activates the new api-based Toolshed installer in the Galaxy's admin panel. This is the first step to properly branch the Toolshed and Galaxy repositories, so that they can maintained individually.

### CODE REVIEW

**Comment by mvdbeek** [comment]
I really appreciate a lot that we do this, but is somebody then going to dedicate some time to improve the current beta install  / future regular install to an acceptable level ? There are some unsolved issues in https://github.com/galaxyproject/galaxy/issues/4076 and there's more that I just didn't have the time to report / diagnose, but that are very obvious if you try using the new interface.

**Comment by dannon** [comment]
@mvdbeek Totally agree, we need to do that first.  I know someone recently said they were going to work on it, but I don't remember who.

**Comment by mvdbeek** [comment]
Hope that wasn't me 😆

**Comment by martenson** [comment]
I did.

**Comment by mvdbeek** [comment]
This looks great to me, but I think something went wrong in merging dev, we now get an additional copy of galaxy.yml.sample when it should be a symlink now.

**Comment by guerler** [comment]
Ah thanks for catching this.

**Comment by mvdbeek** [comment]
I don't think we can remove the "Manage Tools" functionality without replacement. There is now no overview of which repos are installed.

**Comment by mvdbeek** [comment]
Nevermind the last comment -- I just had an empty install database ...

File: lib/galaxy/webapps/galaxy/controllers/admin_toolshed.py [hunk 1]
```diff
@@ -1776,41 +1741,6 @@ def update_to_changeset_revision(self, trans, **kwd):
                                                         message=message,
                                                         status=status))
 
-    @web.expose
-    @web.require_admin
-    def view_tool_metadata(self, trans, repository_id, tool_id, **kwd):
```
Comment (mvdbeek) [inline lib/galaxy/webapps/galaxy/controllers/admin_toolshed.py:1781]: This is still used in the `Manage tools` view, if you click on a tool within a repository you'll get a 404.

**Review by mvdbeek — APPROVED** [review]
Awesome, that's a big step forward!

**Comment by guerler** [comment]
@mvdbeek thanks a lot for the help and review on this project!

**Pull request merged**